tutorials/dynamic-1/dynamic-1.md

```diff
@@ -33,19 +33,12 @@ it addresses conceptual problems for two different types of developers:
 - Experienced kernel developers whose minds are pre-programmed to
       think in terms of "One address space equals one process", and
       begins to introduce the seL4 CSpace vs VSpace model.
-- New kernel developers, for whom the tutorial will prompt them on
-      what to read about.
+- New kernel developers, for whom the tutorial will provide prompts on what to read.
 
 Don't gloss over the globals declared before `main()` -- they're declared
 for your benefit so you can grasp some of the basic data structures.
 
-## Prerequisites
-
-1. [Set up your machine](https://docs.sel4.systems/HostDependencies).
-2. [Hello world](https://docs.sel4.systems/Tutorials/hello-world)
```

Comment:
Should we really remove all these? There is more than one way to do the tutorials, and these tell you which ones you must do before. We could remove the "set up your machine link" that is everywhere, but the other ones I'd leave in.